### ISSUE TITLE
Verify Failures on AtTiny85 -  Again!

### ISSUE THREAD

**Issue by NormanDunbar** [platformio/platformio#800]
Referring back to https://github.com/platformio/platformio/issues/799 which was closed yesterday, somewhat premature I think...and apologies for opening another issue, there's no "request reopen" on the GitHub Issues page. (That I can find anyway).

I followed the instaructions indicated by visiting http://docs.platformio.org/en/stable/platforms/atmelavr.html#upload-using-programmer and my current platformio.ini is as follows:

**The content of `platformio.ini`:**
```ini
[env:attiny85]
platform=atmelavr
board=attiny85
framework=arduino
upload_protocol = usbtiny

[platformio]
src_dir=AtTiny85_Blink
```

_Exactly_ the same happens. Same error message - unable to verify after uploading.

I know it's a PITA when you think we users haven't read the manuals etc, but please don't close this issue until I can at least confirm that it works. Commenting on a closed issue simply loses any further communications.

With the Arduino IDE, I don't even have to "upload using programmer" to get a valid write and verify.

With PlatformIO, until I upgraded, I had no problems uploading - as opposed to programming - with the platformio.ini that I originally posted. Only after the upgrade were there any problems. I'm still at the point of being unable to program any of my AtTiny85 chips.

Changing to the advised upload_protocol had no effect. Still fails to verify the upload.

Sorry. :(

Cheers.

**Comment by ivankravets** [comment]
Please move this issue to https://github.com/platformio/platform-atmelavr/issues
